### PR TITLE
Monitoring Linux VM through DataDog extension

### PR DESCRIPTION
**Pull request by kmouss**
Adding template to enable monitoring through 3rd party DataDog extension solution

### CODE REVIEW

File: datadog-monitoring-linux-vm/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    "parameters": {
+        "newStorageAccountName": {
+            "type": "string",
+                "metadata": {
+                    "description": "Unique Name for the Storage Account where the Virtual Machine's disks will be placed."
+                }
+        },
+        "location": {
+            "type": "string",
+            "defaultValue": "West US",
```
Comment (singhkay) [inline datadog-monitoring-linux-vm/azuredeploy.json]: Please remove the default value

File: datadog-monitoring-linux-vm/azuredeploy.json [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+        "extensionName": "DataDogExtension",
+        "addressPrefix": "10.0.0.0/16",
+        "subnetName": "Subnet",
+        "subnetPrefix": "10.0.0.0/24",
+        "storageAccountType": "Standard_LRS",
+        "publicIPAddressName": "myPublicIP",
+        "publicIPAddressType": "Dynamic",
+        "vmStorageAccountContainerName": "vhds",
+        "vmName": "MyDataDogVM",
+        "vmSize": "Standard_D1",
```
Comment (singhkay) [inline datadog-monitoring-linux-vm/azuredeploy.json]: Please make this a parameter

File: datadog-monitoring-linux-vm/azuredeploy.parameters.json [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "newStorageAccountName": {
+            "value": "unqiueStorageAccount"
+        },
+        "adminUsername": {
+            "value": "userName""
```
Comment (singhkay) [inline datadog-monitoring-linux-vm/azuredeploy.parameters.json]: There's an extra quote

File: datadog-monitoring-linux-vm/metadata.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Deploy a Ubuntu VM with DataDog Monitoring extension.",
```
Comment (singhkay) [inline datadog-monitoring-linux-vm/metadata.json]: Suggestion for name: Ubuntu VM with DataDog monitoring extension

File: datadog-monitoring-linux-vm/log.txt [hunk 5]
```diff
@@ -0,0 +1,447 @@
+info:    Executing command group log show
```
Comment (johngossman) [inline datadog-monitoring-linux-vm/log.txt]: Did you mean to check in this file?

File: datadog-monitoring-linux-vm/README.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+| Name   | Description    |
+|:--- |:---|
+| newStorageAccountName  | Unique DNS Name for the Storage Account where the Virtual Machine's disks will be placed. |
+| location | The location where the Virtual Machine will be deployed |
+| adminUsername  | Username for the Virtual Machine  |
+| adminPassword  | Password for the Virtual Machine  |
+| dnsNameForPublicIP  | Unique DNS Name for the Public IP used to access the Virtual Machine. |
+| ubuntuOSVersion  | The Ubuntu version for deploying the Docker containers. This will pick a fully patched image of this given Ubuntu version. Allowed values: 14.04.2-LTS, 14.04-DAILY, 15.04, 14.10. |
+
```
Comment (johngossman) [inline datadog-monitoring-linux-vm/README.md]: need to document the api-key parameter

**Comment by johngossman** [comment]
We need a little more documentation. For example, how do I actually monitor the output? Is there a url or port I need to hit to get the data?

**Comment by singhkay** [comment]
Closing as not updated for few months. Please feel free to reopen with an update.

**Pull request closed without merging**